### DISCUSSION TITLE
Polar plots in Plotnine?

### DISCUSSION THREAD

**Discussion by RobbesU** [has2k1/plotnine#649] [General]
R ggplot has polar coordinates  ([https://ggplot2.tidyverse.org/reference/coord_polar.html](https://ggplot2.tidyverse.org/reference/coord_polar.html)).
I don't see polar coordinates in plotnine coordinates in the API and cannot find any other reference.
Q: Am i missing them? If not, is there a plan to include them?
Thanks!

**Comment by JBuursma** [comment]
See also https://github.com/has2k1/plotnine/issues/10.

**Reply by RobbesU** [reply]
Thanks @JBuursma . I did a search in this repository on 'polar' but didn't show anything... Must've messed up somehow.
